cmd/swarm/main.go

```diff
@@ -503,7 +497,9 @@ func registerBzzService(bzzconfig *bzzapi.Config, ctx *cli.Context, stack *node.
 		}
 
 		// In production, mockStore must be always nil.
-		return swarm.NewSwarm(ctx, swapClient, bzzconfig, nil)
+		bzz, err := swarm.NewSwarm(ctx, swapClient, bzzconfig, nil)
+		log.Warn("bzz svc", "bzz", bzz)
```

Comment:
I don't think we should have this as a `Warning`. If this is important info, keep if Trace, or Debug, or Info (those terms are a bit overloaded, but this is def not a Warning or an Error)

Reply:
removed